### DISCUSSION TITLE
How to use on:change with FormAutocomplete?

### DISCUSSION THREAD

**Discussion by calionauta** [yesvelte/yesvelte#527] [Q&A] [answered]
I tried to use on:change with FormAutocomplete but it doesn't seem to have worked. 
Is there anything different to do than what we would do with <SELECT>?

**Comment by TheHadiAhmadi** [comment]
Hey @renatocaliari FormAutocomplete doesn't support `change` event, instead you can use `changed` event.

**Reply by calionauta** [reply]
thanks. would it be `on:changed`?

**Reply by calionauta** [reply, marked as answer]
Oh, I got it. I've just reread the doc:

```
<Autocomplete
	items={['a', 'b', 'c', 'd', 'e', 'f']}
	on:changed={() => (changed += 1)}
	placeholder="Change My Value!" />

changed: {changed}
```
